charts/keda-http-operator/crds/http.keda.sh_httpscaledobjects.yaml

```diff
@@ -85,6 +93,10 @@ spec:
                 - port
                 - service
                 type: object
+              targetMetric:
+                description: (optional) Target metric value
```

Comment:
Could you add a bit more explanation here? I think people will get lost on what this is for

Reply:
Thank you, Yeah sure I will add some explanation. As for the naming yeah lets change it if its causing confusion. However I feel `scaleRequestAmount` is a bit counterintuitive since the lower this value the faster it scales. Let me know your thoughts

Reply:
What about `queueMinScaleSize`?

Reply:
That makes more sense but is a bit too long, How about `targetQueueSize` ?

Reply:
The thing about the `target` is that it seems that it'll only scale once it hits this exact number. What about `scaleReqCount`?

Reply:
The reason I used `target` is because I though its relatable to other scaling methods such as CPU for example where the ratio of `current/target ` is considered. I don't understand why we need to have `scale` in the name for this metric. Maybe we could have a poll ? Lets use whatever most people find most intuitive ?

Reply:
I was talking to @arschles today, we came up with `maxParallelRequests`. As this metric is going to be the amount of requests per each app. What do you think?

Reply:
Yes that makes more sense as it is the amount of requests. Let's go with that then.

Reply:
What do you think @zroubalik? I find it a bit technical and was thinking more along the lines of `requestsPerSecond` since, in the future, that is going to be the trigger for SMI (for example)

Reply:
That is another point too, we need to make it simple for end users. I agree with this naming

Reply:
In my opinion `targetQueueSize` feels the most intuitive since the HPA is comparing the `currentQueueSize ` vs `targetQueueSize`

Reply:
I'm +1 to @ajanth97 (he and I are talking about this now). 

Also, this isn't necessarily the number of request per second, or the target number of requests per second. The time window will depend on polling intervals and the semantics of this value are more nuanced. Also, in the SMI world, we might not be using requests per polling interval, so I think we should assume we'll use a different field in the `HTTPScaledObject` for that.

Reply:
Let's go with that :)

Reply:
Frankly, if you do not know how it works using `queue` is very confusing as you are using HTTP traffic and not a queue.

When scaling HTTP, they don't care about the inner workings - They just want it to scale to meet the demand.

Reply:
But if @zroubalik likes the name as well then that's fine for me.

Reply:
@tomkerkhove how about `targetPendingRequests` then? I agree with you RE the `queue` nomenclature, but I think if a user wants to tweak this, then they will be interested enough in inner workings to learn about how the Addon treats pending requests.

Reply:
> @tomkerkhove how about `targetPendingRequests` then? I agree with you RE the `queue` nomenclature, but I think if a user wants to tweak this, then they will be interested enough in inner workings to learn about how the Addon treats pending requests.

I think you might be surprised about how many people will (want) to define their own number, without fully understanding/caring how it works internally.

Name is fine for me.

Reply:
>I think you might be surprised about how many people will (want) to define their own number, without fully understanding/caring how it works internally.

if they do want to do so, and don't understand/care how it works internally, we'll have good documentation (which I'll submit in a follow-up to this PR) to explain it